src/translations/en/basicData.json

```diff
@@ -2,14 +2,19 @@
   "add_employee": "Add employee",
   "add_to_your_person_registry": "Add to your person registry",
   "basic_data": "Basic data",
-  "central_import": "Central import",
+  "central_import": {
+    "central_import":  "Central import",
```

Comment:
klassiker vi har alt for mange plasser :D 
selv om jeg misliker `"central_import.central_import"` har jeg aldri funnet en bedre løsning :(